models/multiphase/d2q9_pf_velocity/Dynamics.c.Rt

```diff
@@ -125,8 +144,27 @@ CudaDeviceFunction vector_t calcGradPhi(){
 		}
 	#else
 		if ((NodeType & NODE_BOUNDARY) == NODE_Wall){
-			gradPhi.x = (PhaseF(1,0) - PhaseF(-1,0))/3.0 + (PhaseF(1,1) - PhaseF(-1,-1) + PhaseF(1,-1) - PhaseF(-1,1))/12.0;
-			gradPhi.y = (PhaseF(0,1) - PhaseF(0,-1))/3.0 + (PhaseF(1,1) - PhaseF(-1,-1) + PhaseF(-1,1) - PhaseF(1,-1))/12.0;
+			// compute gradient with one-sided diffs but first check if neighbours are nans
```

Comment:
Probably the `#ifdef` is in the wrong place, as this is just the same code as previously

Reply:
moved ifdef for outflow conditions after previous calc to remove repeated code.